openmoc/materialize.py

```diff
@@ -2,192 +2,319 @@
 # @file materialize.py
 # @package openmoc.materialize
 # @brief The materialize module provides utility functions to read and write
-#        multi-group materials cross-section data from HDF5 binary file format.
+#        multi-group materials cross section data from HDF5 binary file format.
 # @author William Boyd (wboyd@mit.edu)
-# @date April 23, 2013
-
-# Determine which OpenMOC module is being used. This is important for
-# the materialize module since it must instantiate Material objects
-# with the same floating point precision as that compiled into the
-# openmoc module used in the main Python input script to OpenMOC.
+# @date October 9, 2015
 
 import sys
+import os
+
+import numpy as np
```

Comment:
Adding the import of numpy here seems to imply that numpy is now a hard requirement for OpenMOC, whereas `config.py` has a flag indicating whether OpenMOC should be compiled with numpy typemaps. Do you want to just make numpy a hard requirement for OpenMOC now?